crmsh/report/collect.py

```diff
@@ -431,13 +431,28 @@ def pe_to_dot(pe_file: str) -> None:
         logger.warning("pe_to_dot: %s -> %s failed", pe_file, dotf)
 
 
+def add_the_last_pe_file(file_list: List[str]) -> List[str]:
+    for f in file_list[:]:
+        if f.endswith(".last"):
+            num_str = crmutils.read_from_file(f).strip()
+            if num_str and num_str.isdigit() and int(num_str) > 1:
```

Comment:
> 1 or >= 1 ?

Reply:
Changed.

Does this commit work for you?